GetJobber/Jobber-AppTemplate-RailsAPI
# JOB-60134 Update dependencies in Rails Template App

## Why was this PR created?
To update dependencies
## Quick summary of the changes you made.

- Run `bundle update --conservative`
- Run `rake schema:update`
- Add .idea to .gitignore

# QA

1. Setup and run Rails API
2. Setup and run React APP
3. Make sure they are working

- [x] This has been QA'd by the PR owner
- [ ] This has been QA'd by an approver

# References
